### PR TITLE
Sc 279 create endpoint user fetches tasks

### PR DESCRIPTION
**Pull request by marialobillo**

### CODE REVIEW

File: server/build/src/infrastructure/database/postgresql/PostgreSQLTaskRepository.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const PostgreSQLClient_1 = require("./PostgreSQLClient");
+class PostgreSQLTaskRepository {
+    constructor() {
+        this.client = PostgreSQLClient_1.PostgreSQLClient.getInstance();
+    }
+    async getAll() {
+        const result = await this.client.query('SELECT * FROM tasks');
+        return result.map((task) => new task_1.Task(task));
+    }
+    async create(task) {
```
Comment (AbdelrahmanRadwan) [inline server/build/src/infrastructure/database/postgresql/PostgreSQLTaskRepository.js]: You probably should validate the task first before inserting it to the database, right?

Comment (marialobillo) [reply server/build/src/infrastructure/database/postgresql/PostgreSQLTaskRepository.js]: Yes, you are right, I missed the validation, thank you!

Comment (marialobillo) [reply server/build/src/infrastructure/database/postgresql/PostgreSQLTaskRepository.js]: Done, I got a mistake here I could not save a task in db, now is fixed.

**Comment by marialobillo** [comment]
I also renamed another file, the getAllTasksUseCase.ts

**Review by AbdelrahmanRadwan — APPROVED** [review]

**Pull request merged**